src/Tribe/App_Shop.php

```diff
@@ -77,11 +79,25 @@ public function add_toolbar_item() {
 		}
 
 		/**
+		 * Registers the plugin assets
+		 */
+		protected function register_assets() {
+			tribe_assets(
+				Tribe__Main::instance(),
+				array(
+					array( 'app-shop-css', 'app-shop.css' ),
+					array( 'app-shop-js', 'app-shop.js' ),
+				),
+				'admin_print_styles-' . $this->admin_page
```

Comment:
@elimn, @barryhughes mentioned in Slack that `$this->admin_page` is null. If that was set to the correct value, perhaps all would be right in the world?